content/en/docs/contribute/style/style-guide.md

```diff
@@ -263,6 +263,20 @@ The output is:
 **Warning:** Beware.
 {{< /warning >}}
 
+### Katacoda Embedded Live Environment
+
+This button lets users run Minikube in their browser.
```

Comment:
I would add a little more documentation and description here. Specifically, that this is the Katacoda [Terminal Panel](https://www.katacoda.com/embed/panel) configured to run `minikube start`, and it lets users run Minikube as a panel in their browser so they can follow along while they read the documentation.

Reply:
@chenopis done. I've made a proper entry to the style guide.